src/index.html

```diff
@@ -11,12 +11,97 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&amp;display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#1"
+              class="nav__link nav__link-active is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#2"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#3"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              href="#1"
              class="nav__link nav__link-active is-active"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="#2"
              class="nav__link"
            >
              Samsung
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="#3"
              class="nav__link"
            >
              Smartphones
            </a>
          </li>
```

Add spaces between multiple siblings everywhere in code below